core/client/src/main/java/alluxio/client/block/BlockStoreContext.java

```diff
@@ -61,13 +62,8 @@
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   private BlockMasterClientPool mBlockMasterClientPool;
 
-  /**
-   * A map from the worker's address to its client pool. Guarded by
-   * {@link #initializeLocalBlockWorkerClientPool()} for client acquisition. There is no guard for
-   * releasing client, and client can be released anytime.
-   */
-  private final Map<WorkerNetAddress, BlockWorkerClientPool> mLocalBlockWorkerClientPoolMap =
-      new ConcurrentHashMap<>();
+  private static final ConcurrentHashMapV8<InetSocketAddress, BlockWorkerThriftClientPool>
```

Comment:
SG. not necessary to have "A map from the worker's address to its client pool. " as you said, this is obvious.